python/_restclient/hydrotools/_restclient/_restclient.py

```diff
@@ -1,211 +1,231 @@
-"""
-Generic REST Api client usable/inheritable for general usage and adaptable to more
-specific use cases.
+#!/usr/bin/env python3
 
-# TODO: Add more information about the class here
-"""
+from typing import Any, List, Union
+import asyncio
+from aiohttp.client_reqrep import ClientResponse
+from aiohttp_client_cache.response import CachedResponse
+from aiohttp_client_cache import SQLiteBackend
+import aiohttp
+import forge
+import atexit
 
-from typing import Any, Dict, List, Union
-from collections.abc import Iterable
+# local imports
+from .async_client import ClientSession
+from .async_helpers import AsyncToSerialHelper
+from .urllib import Url
+from ._restclient_sigs import GET_SIGNATURE, MGET_SIGNATURE
 
-import requests
-import requests_cache
-from urllib3.util.retry import Retry
+__all__ = ["RestClient"]
 
 
-class RestClient:
-    """Provides various methods for constructing requests, retrieving data, and
-    parsing responses from the NWIS IV Service.
+class RestClient(AsyncToSerialHelper):
+    """
+    Class adept for supporting and simplifying RESTful client libraries and retrieval
```

Comment:
I've noticed a lot of subjective terms in the documentation that I'm not sure help describe the code's function. Is it possible to summarize the inputs and outputs to a method without using words like "adept" or "robust"? At least at this stage, I'm not sure these are claims we can back-up with data. Some subjective terms are self-evident, like "simple" or "convenient," especially with regard to client tools. But I wonder if we keep the subjective descriptions in the README and leave docstrings a bit more sterile.

Reply:
Certainly and thanks for the input. I agree that a more relaxed lexicon is more suitable for documentation.